hw/ip/csrng/doc/checklist.md

```diff
@@ -16,15 +16,15 @@ All checklist items refer to the content in the [Checklist.]({{< relref "/doc/pr
 
 Type          | Item                           | Resolution  | Note/Collaterals
 --------------|--------------------------------|-------------|------------------
-Documentation | [SPEC_COMPLETE][]              | Not Started | [CSRNG Design Spec]({{<relref "hw/ip/csrng/doc" >}})
-Documentation | [CSR_DEFINED][]                | Not Started |
-RTL           | [CLKRST_CONNECTED][]           | Not Started |
-RTL           | [IP_TOP][]                     | Not Started |
-RTL           | [IP_INSTANTIABLE][]            | Not Started |
-RTL           | [PHYSICAL_MACROS_DEFINED_80][] | Not Started |
-RTL           | [FUNC_IMPLEMENTED][]           | Not Started |
-RTL           | [ASSERT_KNOWN_ADDED][]         | Not Started |
-Code Quality  | [LINT_SETUP][]                 | Not Started |
+Documentation | [SPEC_COMPLETE][]              | Done        | [CSRNG Design Spec]({{<relref "hw/ip/csrng/doc" >}})
+Documentation | [CSR_DEFINED][]                | Done        |
+RTL           | [CLKRST_CONNECTED][]           | Done        |
+RTL           | [IP_TOP][]                     | Done        |
+RTL           | [IP_INSTANTIABLE][]            | Done        |
```

Comment:
Have you done any trials to see if the CSRNG module can be instantiated in the toplevel (e.g. a work-in-progress PR somewhere)?

Reply:
CSRNG has been instantiated as part of https://github.com/lowRISC/opentitan/pull/4211, which has been merged a few hours ago.

Reply:
The PR #4211 which just got merged did all of the entropy complex blocks (csrng, edn, entropy_src), so yes, it is already integrated.

Reply:
Wow, I must be blind and utterly confused (here and on the lint one). Sorry for the noise.